### PR TITLE
Yield accounts

### PR DESCRIPTION
**Pull request by HarleyB123**
takes about 10 seconds for the dashboard page to load

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 from fastapi import FastAPI, Form, Request
+from fastapi.responses import StreamingResponse
```
Comment (HarleyB123) [inline main.py:2]: I don't need this, will remove

File: storage.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from azure.mgmt.storage import StorageManagementClient
+from azure.mgmt.subscription import SubscriptionClient
+
+from auth import CREDENTIALS
+
+
+def list_subscriptions():
```
Comment (markusbnet) [inline storage.py:7]: I'm thinking we should have this as a separate file. Maybe merge it with authentication. We will end up reusing it everywhere.

Comment (markusbnet) [reply storage.py:7]: Approving still 👍

**Review by markusbnet — APPROVED** [review]
Lavely!

**Pull request merged**